mmv1/products/datastream/api.yaml

```diff
@@ -369,3 +367,967 @@ objects:
+                                        Column length.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: 'precision'
+                                      description: |
+                                        Column precision.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: 'scale'
+                                      description: |
+                                        Column scale.
+                                    - !ruby/object:Api::Type::Boolean
+                                      name: 'primaryKey'
+                                      description: |
+                                        Whether or not the column represents a primary key.
+                                    - !ruby/object:Api::Type::Boolean
+                                      name: 'nullable'
+                                      description: |
+                                        Whether or not the column can accept a null value.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: 'ordinalPosition'
+                                      description: |
+                                        The ordinal position of the column in the table.
+              - !ruby/object:Api::Type::String
+                name: 'replicationSlot'
+                required: true
+                description: |
+                  The name of the logical replication slot that's configured with the pgoutput plugin.
+              - !ruby/object:Api::Type::String
+                name: 'publication'
+                required: true
+                description: |
+                  The name of the publication that includes the set of all tables that are defined in the stream's includeObjects.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'destinationConfig'
+        required: true
+        description: |
+          Destination connection profile configuration.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'destinationConnectionProfile'
+            required: true
+            description: |
+              Destination connection profile resource. Format: projects/{project}/locations/{location}/connectionProfiles/{name}
+          - !ruby/object:Api::Type::NestedObject
+            name: 'gcsDestinationConfig'
+            exactly_one_of:
+              - destination_config.0.gcs_destination_config
+              - destination_config.0.bigquery_destination_config
+            description: |
+              A configuration for how data should be loaded to Cloud Storage.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'path'
+                description: |
+                  Path inside the Cloud Storage bucket to write data to.
+              - !ruby/object:Api::Type::String
```

Comment:
This is an Integer - https://source.corp.google.com/piper///depot/google3/google/cloud/datastream/v1/datastream_resources.proto;rcl=486920491;l=861